models/reps.js

```diff
@@ -0,0 +1,18 @@
+// data for reps
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const repSchema = new mongoose.Schema({
+    reps: {type: Number, min: 1, max: 100, required: true}
+ }); 
+
+ const findReps = module.exports = mongoose.model('reps', repSchema);
+
+
+ // display 
+ findReps.
+    find().
+    limit(100).
+    select('reps'); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
    select("reps"); 
```